frontend/src/index.scss

```diff
@@ -0,0 +1 @@
+
```

Comment:
Is it common to have an empty file like this one?

Reply:
I just added it to outline file structure, to confirm it with Sasha mostly. So yes, perhaps there will be some global styles and maybe reimports of local styles, we will see.